### PR TITLE
notmuch 0.31

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
license file (`COPYING`)
```
Notmuch is free software.

You can redistribute it and/or modify it under the terms of the GNU
General Public License as published by the Free Software Foundation,
either version 3 of the License, or (at your option) any later
version.

This program is distributed in the hope that it will be useful, but
WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program, (in the COPYING-GPL-3 file in this
directory). If not, see https://www.gnu.org/licenses/

```

File: Formula/notmuch.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Notmuch < Formula
   desc "Thread-based email index, search, and tagging"
   homepage "https://notmuchmail.org/"
-  url "https://notmuchmail.org/releases/notmuch-0.30.tar.xz"
-  sha256 "5e3baa6fe11d65c67e26ae488be11b320bae04e336acc9c64621f7e3449096fa"
+  url "https://notmuchmail.org/releases/notmuch-0.31.tar.xz"
+  sha256 "571fa0e1539c86612b1f2b2c80a398e08ecfef52e27ef7e48cf8e3b84fa18394"
+  license "GPL-3.0-or-later"
```
Comment (claui) [inline Formula/notmuch.rb:6]: I suppose this formula doesn’t use the part of the tree with the Go wrapper [licensed under LGPL 2.1](https://git.notmuchmail.org/git?p=notmuch;a=blob;f=contrib/go/LICENSE;h=4362b49151d7b34ef83b3067a8f9c9f877d72a0e;hb=HEAD)?

**Comment by SMillerDev** [comment]
Needs more license info

**Comment by chenrui333** [comment]
I will put a separate PR for the license stuff, and only have this PR about the version upgrade.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**